web/plugin-manager/src/helpers/statusHelpers.ts

```diff
@@ -0,0 +1,16 @@
+export const statusHelper = (status: string) => {
```

Comment:
in general when you rturn a boolean you should use a isXXX or hasXXX.

Reply:
changed for isStatusReady